manifests/istio-control/istio-autoinject/files/injection-template.yaml

```diff
@@ -388,23 +373,6 @@ template: |
   - emptyDir:
       medium: Memory
     name: istio-envoy
-  {{- if .Values.global.sds.enabled }}
-  - name: sds-uds-path
-    hostPath:
-      path: /var/run/sds
-  - name: istio-token
-    projected:
```

Comment:
How could it work if we don't have the istio-token here? I searched in the file but couldn't found a block to enable istio-token when istiod is enabled.